easy_my_coop/models/company.py

```diff
@@ -92,6 +92,9 @@ def _compute_base_logo(self):
         translate=True,
         help="Text to display aside the checkbox to approve financial risk.",
     )
+    send_certificate_email = fields.Boolean(string="Send certificate email", default=True)
+    send_confirmation_email = fields.Boolean(string="Send confirmation email", default=True)
+    send_capital_release_email = fields.Boolean(string="Send Capital Release email", default=True)
```

Comment:
could you format the code to comply with the actual formating ?

Reply:
Of course! Done!

Reply:
thank you